Exception::__set_state(array(
   'message' => '',
   'string' => 'Exception in %sbug73350.php:%d
Stack trace:
#0 {main}',
   'code' => 0,
   'file' => '%sbug73350.php',
   'line' => %d,
   'trace' => 
  array (
  ),
   'previous' => NULL,
))